lib/Service/MailService.php

```diff
@@ -102,6 +107,12 @@ public function sendTokenByMail(Registration $registration): void {
 			htmlspecialchars($body . ' ' . $this->l10n->t('Click the button below to continue.')),
 			$body
 		);
+		
+		// if the parameter is set through the settings panel add to body text
+		$email_verification_hint = $this->config->getAppValue('registration', 'email_verification_hint');
+ 		if (!empty($email_verification_hint)) {
+				$template->addBodyText($email_verification_hint);
+		}; 
```

Comment:
```suggestion
		if (!empty($email_verification_hint)) {
			$template->addBodyText($email_verification_hint);
		};
```

This should fix the failing CS test